### PR TITLE
V1.9.0 patchset

### PR DESCRIPTION
**Pull request by rphillips**

### CODE REVIEW

File: cluster/images/hyperkube/Makefile [hunk 1]
```diff
@@ -20,13 +20,15 @@
 REGISTRY?=gcr.io/google-containers
 ARCH?=amd64
 HYPERKUBE_BIN?=_output/dockerized/bin/linux/$(ARCH)/hyperkube
+CALICO_RELEASE=v1.11.0
```
Comment (colemickens) [inline cluster/images/hyperkube/Makefile]: There's a `v1.11.1` available.

File: cluster/images/hyperkube/Makefile [hunk 2]
```diff
@@ -51,4 +53,11 @@ ifeq ($(ARCH),amd64)
 	gcloud docker -- push ${REGISTRY}/hyperkube:${VERSION}
 endif
 
-.PHONY: build push all
+cni:
+	mkdir -p ${TEMP_DIR}/cni-bin/bin
+	curl -sSL --retry 5 https://github.com/containernetworking/cni/releases/download/${CNI_RELEASE}/cni-${ARCH}-${CNI_RELEASE}.tgz | tar -xz -C ${TEMP_DIR}/cni-bin/bin
+	curl -sSL --retry 5 https://github.com/containernetworking/plugins/releases/download/${CNI_RELEASE}/cni-plugins-${ARCH}-${CNI_RELEASE}.tgz | tar -xz -C ${TEMP_DIR}/cni-bin/bin
+	curl -sSL --retry 5 -o ${TEMP_DIR}/cni-bin/bin/calico https://github.com/projectcalico/calico-cni/releases/download/${CALICO_RELEASE}/calico
```
Comment (colemickens) [inline cluster/images/hyperkube/Makefile]: GitHub redirects, but they've renamed this repo to `cni-plugin`

Comment (rphillips) [reply cluster/images/hyperkube/Makefile]: fixed.

File: cluster/images/hyperkube/Makefile (hunk 1 above)
Comment (rphillips) [reply cluster/images/hyperkube/Makefile]: fixed

**Review by colemickens — APPROVED** [review]
lgtm

**Pull request merged**